css/terminal7.css

```diff
@@ -115,7 +115,7 @@ a {
     background-color: var(--background);
     padding: 2%;
     position: fixed;
-    bottom: 90px;
+    bottom: 180px;
```

Comment:
This class is used elsewhere and this could change the UI. better to override the bottom on keys-help